index.fr.md

```diff
@@ -0,0 +1,226 @@
+
+Note : en plus de détecter des problèmes d'accessibilité, l'évaluation menée avec des utilisateurs handicapés révèlent des problèmes généraux d'utilisation qui touchent tous les utilisateurs, y compris ceux qui ne se trouvent *pas* en situation de handicap.
+
+## Analyse des problèmes d'accessibilité {#analyz}
+
+L'accessibilité Web dépend de l'association [des différents composants du développement Web avec l'interaction](/fundamentals/components/), dont les navigateurs Web, les technologies d'assistance, et le contenu Web.
+
+Les problèmes d'accessibilité peuvent être causés par un ou plusieurs composants. Par exemple, si un utilisateur qui ne peut pas utiliser une souris n'arrive pas à accéder aux fonctionnalités clavier des technologies d'assistance, il peut y avoir plusieurs explications :
+
+-   le développeur n'a pas balisé/codé correctement la page Web, ou
+-   le navigateur ou le lecteur multimédia ne lit pas correctement le code, ou
+-   la technologie d'assistance ne lit pas correctement le code, ou
+-   l'utilisateur ne sait pas comment utiliser le navigateur ou le lecteur multimédia ou les fonctionnalités accessibles au clavier des technologies d'assistance, ou
+-   la page n'est pas correment conçue et il s'agit d'un problème général d'utilisation pour tous les utilisateurs, y compris ceux qui ne se trouvent *pas* en situation de handicap.
+s
+## Associez l'évaluation des utilisateurs aux standards {#stdstoo}
+
+Inclure les utilisateurs handicapés dans l'évaluation présente de nombreux avantages. Ceci dit, cela ne suffit pas à déterminer l'accessibilité d'un site Web. Associez l'inclusion des utilisateurs à l'[évaluation de la conformité aux WCAG](/test-evaluate/conformance/) afin de garantir l'accessibilité pour différents types de handicaps et de situations.
+
+## Tirez des conclusions et créez un rapport {#drawing}
+
+Faites attention si vous tirez des conclusions d'évaluations ou d'études limitées. Les résultats obtenus avec deux personnes handicapées ne peuvent être généralisés et appliqués à toutes les personnes dans la même situation de handicap ou présentant d'autres handicaps. Retrouvez plus d'informations dans la section [Attention](#caution) ci-dessous.
+
+Incluez dans votre rapport le cadre de votre étude et les paramètres de l'évaluation, tels que les méthodes de test et les caractéristiques des utilisateurs. Par exemple, si une étude a inclu uniquement des personnes aveugles, le rapport devra préciser que celle-ci n'a pas évalué la conformité aux standards d'accessibilité et qu'elle ne s'applique pas à toutes les personnes handicapées. Bien que les études de petite envergure peuvent souvent fournir des informations utiles, elles ne sont pas assez solides pour fournir une signification statistique.
```

Comment:
- Scope : Maybe "périmètre" (what was the extent of the research) is more precise here, considering the rest of the sentence is focusing on the number of people involved. "Cadre" seems more broad : context, hypothesis, etc.